fine_tune.py

```diff
@@ -461,12 +470,13 @@ def fn_recursive_set_mem_eff(module: torch.nn.Module):
         train_util.save_sd_model_on_train_end(
             args, src_path, save_stable_diffusion_format, use_safetensors, save_dtype, epoch, global_step, text_encoder, unet, vae
         )
-        print("model saved.")
+        logger.info("model saved.")
 
 
 def setup_parser() -> argparse.ArgumentParser:
     parser = argparse.ArgumentParser()
 
+    add_logging_arguments(parser)  # モジュール指定がないのがちょっと気持ち悪い / bit weird that this does not have module prefix
```

Comment:
Moving `add_logging_arguments()` and `setup_logging()` from `library/utils.py` to `library/logger_utils/__init__.py` is an option.

The call will be like this.
```python
logger_utils.add_logging_arguments(parser)
```